documentation/docs/examples/cad-transform-override.mdx

```diff
@@ -0,0 +1,87 @@
+---
+
+import { DemoWrapper } from '@site/src/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Reveal supports overriding transforms of individual objects by providing [tree indices to identify 3D nodes.](../concepts.md)
+
+Note that any transformation override is applied with respect to the THREE.js world coordinate space.
+
+## Examples
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+### Setting node transform
+
+Setting the transform of a given node is done by `Cognite3DModel.setNodeTransformByTreeIndex` and takes a tree index,
+position, rotation and a flag if this transformation also should be applied to children of the given node.
+
+
+
+```jsx runnable
+const position = new THREE.Vector3(10, 10, 0);
+const rotation = new THREE.Euler(0, 0, 0);
+model.setNodeTransformByTreeIndex(532261, {position, rotation}, true);
+```
+
+### Remove node transform
+
+Removing the transformation is done with `Cognite3DModel.resetNodeTransformByTreeIndex` and takes the given tree index,
+and a flag that determines if the node transform reset also should be applied to any child nodes of the given tree index.
+
+```jsx runnable
+model.resetNodeTransformByTreeIndex(532261, true);
+```
+
+### Animations
+
+Creating an animations is a matter of applying transformations multiple times. As stated above, any transformation (position and / or rotation),
+is applied in the global world coordinate system of THREE.js. This means that if you want to rotate a model around a given point (and not around the world origin),
+you first need to transform the model to that given point, then rotate, then transform back into the world coordinate space. An example of this is done below to create an animation
+of the crane rotating. First we get the center of the bounding box for node with id `533010` which is close to the base of the crane.
+Then we set up an interval which triggers a function that transforms the entire crane (`532261`): first to the base of the crane, then rotates the model,
+then transforms it back, thus giving us the wanted rotation. Keep in mind that THREE.js uses column matrix notation, such that
+the following statement: 
+
+```js
+  const matrixOverride = modelToWorld.multiply(rotationMatrix.multiply(worldToModel));
+```
+
+first applies `worldToModel`, then `rotationMatrix` and finally `modelToWorld`.
+
+```jsx runnable
+model.getBoundingBoxByTreeIndex(533010).then(boundingBox => {
+  let counter = 0;
+
+  const center = boundingBox.getCenter(new THREE.Vector3());
+
+  const modelToWorld = new THREE.Matrix4();
+  const worldToModel = new THREE.Matrix4();
+  const rotationMatrix = new THREE.Matrix4();
+
+  const rotation = new THREE.Euler();
+  const position = new THREE.Vector3();
+
+  const intervalID = setInterval(function () {
+    modelToWorld.setPosition(center);
+    worldToModel.getInverse(modelToWorld.clone());
+
+    const newRotation = new THREE.Euler(0, -Math.PI / 300 * counter, 0);

```

Comment:
I think we should avoid using a counter for determining "time" and use e.g. `THREE.Clock` instead. Not because it's really important here, but as a little pointer for people reading the doc